doc/source/whatsnew/v0.16.0.txt

```diff
@@ -106,6 +106,51 @@ Enhancements
 - ``tseries.frequencies.to_offset()`` now accepts ``Timedelta`` as input (:issue:`9064`)
 
 - ``Timedelta`` will now accept nanoseconds keyword in constructor (:issue:`9273`)
+- Added :meth:`SparseSeries.to_coo` and :meth:`SparseSeries.from_coo` methods
```

Comment:
let's add this in its own sub-section (e.g. put ``-----``) underneath the title, then you don't have to indent at all